package Lyekka.pb;
option optimize_for = SPEED;

import "lyekka.proto";

extend TreeRef {
  optional bytes tree_sha_ext = 1000;
}

extend Part {
  optional bytes part_sha_ext = 1000;
}